sample-project/src/test/kotlin/io/cucumber/kotlin/LambdaStepDefinitions.kt

```diff
@@ -11,6 +11,16 @@ var lastInstance: LambdaStepDefinitions? = null
 class LambdaStepDefinitions : En {
 
     init {
+         ParameterType("stringList", "(\"([^\"]+)\"(\\s*(([,]?\\s*)|(and\\s?))\"[^\"]+\")*)") { strings: String ->
+            val compile: Pattern = Pattern.compile("\\s*(,|and)?\\s*\"([^\"]+)\"")
+            val matcher: Matcher = compile.matcher(strings)
+            val result: MutableList<String> = ArrayList()
+            while (matcher.find()) {
+                result.add(matcher.group(2))
+            }
+            result
+        }
```

Comment:
The Pattern and Matcher classes are missing from the imports. The Java classes are used for regex and matching. It would be nice if the Kotlin equivalent could be used as an example.